SU-SWS/stanford_bean_types
# Hsdo 113 more icons

# READY FOR REVIEW

# Summary
- Extended the social media block to allow different icons by using the font awesome field type. Also allows the user to change the order and change the color depending on their preferences. All fields are required to force the user to filling out all necessary data.

# Needed By (Date)
- 4/12

# Steps to Test
1. Make note of the appearance & urls of a social connect block in the footer on a desired page
2. Checkout branch & checkout branch on [stanford_field_formatters](https://github.com/SU-SWS/stanford_field_formatters/pull/3)
3. Run database updates
4. Review the social connect block from earlier to validate it looks similar and has the same links.
5. edit that block and manipulate as desired (change order, icons, colors etc)
6. verify it works as expected.

# Associated Issues and/or People
- [stanford_field_formatters](https://github.com/SU-SWS/stanford_field_formatters/pull/3)
